src/routes/spellbook/+page.svelte

```diff
@@ -12,10 +12,24 @@
     import { cubicInOut } from "svelte/easing";
     import { flip } from "svelte/animate";
     import type { Spell } from "$lib/types/spell";
+    import DropDownClasses from "$lib/components/DropDownClasses.svelte";
+    import DropDownLevels from "$lib/components/DropDownLevels.svelte";
+    import { SlideToggle } from '@skeletonlabs/skeleton';
 
     let ownerInput = "";
     let searchInput = "";
     let spells: Spell[] = [];
+    let selectedLevel = "all";
+    let selectedClass = "all";
+    let originalSpellsOrder: Spell[] = [];
+    let isSortingByAscending = false;
+    let a5e: boolean = false;
+    let groupedSpells: Record<string, Spell[]> = {};
+
+    $: {
+        const spellsInCurrentBook = $spellbookStore[$activeSpellbookIndex]?.spells || [];
+        groupSpellsByLevel();
+    }
```

Comment:
Variablen `spellsInCurrentBook` bliver ikke brugt. Den kan fjernes og udtrykket kan rettes til nedenstående. Så reagerer den på en ændring `$spellbookStore.length`:

```suggestion
    $: if ($spellbookStore.length) {
        groupSpellsByLevel();
    }
```

Reply:
Virker ikke